projects/batfish/src/test/java/org/batfish/bdp/AbstractRibTest.java

```diff
@@ -0,0 +1,154 @@
+package org.batfish.bdp;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.emptyIterableOf;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import org.batfish.datamodel.Ip;
+import org.batfish.datamodel.Prefix;
+import org.batfish.datamodel.StaticRoute;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test the AbstractRib tree logic. To avoid worrying about route preference comparisons, testing
+ * with StaticRoutes and StaticRib. All static routes will be stored in the RIB, without eviction
+ * based on preference.
+ */
+public class AbstractRibTest {
+  private AbstractRib<StaticRoute> _rib;
+  private StaticRoute _mostGeneralRoute;
+
+  @Before
+  public void setupEmptyRib() {
+    _rib = new StaticRib(null);
+    _mostGeneralRoute = new StaticRoute(new Prefix("0.0.0.0/0"), Ip.ZERO, null, 0, 0);
+  }
+
+  @Test
+  public void testRibConstructor() {
+    // Test: the setupEmptyRib() fixture
```

Comment:
Drop these comments -- structuring the test in a logical order with, optionally, whitespace between sections is good enough :)